src/h5web/App.test.tsx

```diff
@@ -81,79 +81,95 @@ test('toggle explorer sidebar', async () => {
 
-  const childGroupBtn = getByRole('treeitem', { name: '1_cormap' });
+  const childGroupBtn = getByRole('treeitem', { name: 'empty_group' });
   expect(childGroupBtn).toHaveAttribute('aria-selected', 'false');
   expect(childGroupBtn).toHaveAttribute('aria-expanded', 'false');
 
-  // Expand `1_cormap` child group
+  // Expand `empty_group` child group
   fireEvent.click(childGroupBtn);
 
   expect(groupBtn).toHaveAttribute('aria-selected', 'false');
   expect(groupBtn).toHaveAttribute('aria-expanded', 'true');
   expect(childGroupBtn).toHaveAttribute('aria-selected', 'true');
   expect(childGroupBtn).toHaveAttribute('aria-expanded', 'true');
 
-  // Collapse `1_cormap` child group
+  // Collapse `empty_group` child group
   fireEvent.click(childGroupBtn);
 
   expect(childGroupBtn).toHaveAttribute('aria-selected', 'true');
   expect(childGroupBtn).toHaveAttribute('aria-expanded', 'false');
 
-  // Select `entry_0000` group
+  // Select `entities` group
   fireEvent.click(groupBtn);
 
   expect(groupBtn).toHaveAttribute('aria-selected', 'true');
   expect(groupBtn).toHaveAttribute('aria-expanded', 'true'); // remains expanded as it wasn't previously selected
   expect(childGroupBtn).toHaveAttribute('aria-selected', 'false');
 
-  // Collapse `entry_0000` group
+  // Collapse `entities` group
   fireEvent.click(groupBtn);
 
-  expect(queryByRole('treeitem', { name: '1_cormap' })).not.toBeInTheDocument();
+  expect(
+    queryByRole('treeitem', { name: 'empty_group' })
+  ).not.toBeInTheDocument();
   expect(groupBtn).toHaveAttribute('aria-selected', 'true');
   expect(groupBtn).toHaveAttribute('aria-expanded', 'false');
 });
 
-test('visualise a scalar dataset', async () => {
-  const { findByRole, getByRole, getByText } = renderApp();
+test('visualise scalar dataset', async () => {
+  const { findByRole, getByRole, findByText, getByText } = renderApp();
 
-  // Select the scalar dataset entry_0000/1_cormap/version
-  fireEvent.click(await findByRole('treeitem', { name: 'entry_0000' }));
-  fireEvent.click(getByRole('treeitem', { name: '1_cormap' }));
-
-  const datasetBtn = getByRole('treeitem', { name: 'version' });
+  fireEvent.click(await findByRole('treeitem', { name: 'entities' }));
+  const datasetBtn = getByRole('treeitem', { name: 'scalar_int' });
   expect(datasetBtn).toBeVisible();
 
   fireEvent.click(datasetBtn);
   expect(datasetBtn).toHaveAttribute('aria-selected', 'true');
-  expect(datasetBtn).not.toHaveAttribute('aria-expanded'); // A dataset cannot be expanded
+  expect(datasetBtn).not.toHaveAttribute('aria-expanded'); // a dataset cannot be expanded
 
   const scalarTab = await findByRole('tab', { name: 'Scalar' });
   expect(scalarTab).toBeVisible();
   expect(scalarTab).toHaveAttribute('aria-selected', 'true');
+  expect(getByText('0')).toBeVisible();
+
+  fireEvent.click(getByRole('treeitem', { name: 'scalar_str' }));
+  expect(await findByText('foo')).toBeVisible();
+});
+
+test('inspect scalar dataset', async () => {
```

Comment:
Just moving this test up and "simplifying" it by inspecting a scalar dataset rather than a 1D dataset.